### ISSUE TITLE
steal-tools build options page footer broken 

### ISSUE THREAD

**Issue by m-mujica** [stealjs/stealjs#59]
![screen shot 2017-08-04 at 16 30 12](https://user-images.githubusercontent.com/724877/28989476-bfe9cc0e-7932-11e7-8d8e-ae31cb789304.png)

**Comment by m-mujica** [comment]
the generated html has the docObject declared in a script

```html
var docObject = {  /*massive object*/ };
```

An html snippet in the buildOptions page is causing the browser to throw an invalid token, I guess some unescaped symbol or something. The weird thing is that if I paste the object in the terminal the JS parser does not complain.

As a quick fix I removed the tags and re-deployed the docs from my branch, I made sure all the other packages where up to date, hopefully I did not introduce any regressions.

I'll take a deeper look next week.

![screen shot 2017-08-04 at 17 57 15](https://user-images.githubusercontent.com/724877/28990834-6e41ddfe-793e-11e7-9b2d-5092cef40721.png)
